apollo-normalized-cache/src/main/java/com/apollographql/apollo/cache/normalized/lru/LruNormalizedCache.kt

```diff
@@ -21,21 +21,21 @@ class LruNormalizedCache internal constructor(evictionPolicy: EvictionPolicy) :
 
   private val lruCache: Cache<String, Record> =
       CacheBuilder.newBuilder().apply {
-        if (evictionPolicy.maxSizeBytes().isPresent) {
-          maximumWeight(evictionPolicy.maxSizeBytes().get()).weigher(
+        if (evictionPolicy.maxSizeBytes != null) {
+          maximumWeight(evictionPolicy.maxSizeBytes).weigher(
               Weigher { key: String, value: Record ->
                 key.toByteArray(Charset.defaultCharset()).size + value.sizeEstimateBytes()
               }
           )
         }
-        if (evictionPolicy.maxEntries().isPresent) {
-          maximumSize(evictionPolicy.maxEntries().get())
+        if (evictionPolicy.maxEntries != null) {
+          maximumSize(evictionPolicy.maxEntries)
         }
-        if (evictionPolicy.expireAfterAccess().isPresent) {
-          expireAfterAccess(evictionPolicy.expireAfterAccess().get(), evictionPolicy.expireAfterAccessTimeUnit().get())
+        if (evictionPolicy.expireAfterAccess != null) {
+          expireAfterAccess(evictionPolicy.expireAfterAccess, evictionPolicy.expireAfterAccessTimeUnit!!)
```

Comment:
@tasomaniac I had to use force unwrap here since `expireAfterAccess()` expects a `Nonnull`.

Given that the existing implementation does not check with `isPresent()` for `expireAfterAccessTimeUnit`, it the force unwrap here okay?

Reply:
Yeah, I think this is fine. It's the exact same behavior